WcaOnRails/spec/mailers/competitions_mailer_spec.rb

```diff
@@ -74,6 +74,23 @@
     end
   end
 
+  describe "submit_report_nag" do
+    let(:competition) { FactoryGirl.create(:competition, name: "Peculiar Comp 2016") }
+    let(:mail) { CompetitionsMailer.submit_report_nag(competition) }
+
+    it "renders the headers" do
+      expect(mail.subject).to eq "Peculiar Comp 2016 Delegate Report"
+      expect(mail.to).to match_array competition.delegates.pluck(:email)
+      expect(mail.cc).to eq ["results@worldcubeassociation.org", "board@worldcubeassociation.org"]
```

Comment:
I think we should CC their senior delegate (if they have one) as well here.

Did you discuss with @Luis-J-Ianez about if the WRT should be CC-ed in this email? To me, this feels like something between the delegate(s), their senior delegates, and the Board.